DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,192 @@
+}
+
+@section NavMenuItems {
+  <partial name="/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(@Model.CustomisationNameSuffix), nameof(@Model.Password), nameof(@Model.NotificationEmails), nameof(@Model.TutCompletionThreshold), nameof(@Model.DiagCompletionThreshold) }" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit course details</h1>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post"
+          novalidate
+          asp-controller="ManageCourse"
+          asp-action="SaveCourseDetails"
+          asp-route-customisationId="@customisationId">
+
+      <input type="hidden" asp-for="CentreId" />
+      <input type="hidden" asp-for="ApplicationId" />
+      <input type="hidden" asp-for="CustomisationName" />
+      <input type="hidden" asp-for="PostLearningAssessment" />
+      <input type="hidden" asp-for="DiagAssess" />
+
+      <div class="nhsuk-grid-row nhsuk-u-margin-bottom-5">
+
+        <div class="nhsuk-grid-column-one-quarter nhsuk-heading-s">
+          <label class="nhsuk-heading-s nhsuk-u-font-weight-normal nhsuk-u-margin-bottom-0">
+            Course name
+          </label>
+          <div class="nhsuk-u-margin-top-2">
+            @Model.CustomisationName -
+          </div>
+        </div>
+        <div class="nhsuk-grid-column-one-half nhsuk-u-margin-top-2">
+          <vc:text-input asp-for="@nameof(Model.CustomisationNameSuffix)"
+                         id="CustomisationNameSuffix"
+                         label="Add on to course name (optional)"
+                         populate-with-current-value="true"
+                         type="text"
+                         spell-check="true"
+                         autocomplete=""
+                         hint-text=""
+                         css-class="nhsuk-u-width-three-quarters nhsuk-u-font-weight-normal" />
+        </div>
+      </div>
+
+      <div class="nhsuk-form-group nhsuk-u-margin-bottom-5">
+        <fieldset class="nhsuk-fieldset">
+          <div class="nhsuk-checkboxes nhsuk-checkboxes--conditional">
+
+            <div class="nhsuk-u-margin-bottom-3">
+              <div class="nhsuk-checkboxes__item">
+                <input class="nhsuk-checkboxes__input"
+                       id="PasswordProtected"
+                       name="PasswordProtected"
+                       asp-for="PasswordProtected"
+                       type="checkbox"
+                       aria-controls="@passwordId"
+                       aria-expanded="@(Model.PasswordProtected ? "true" : "false")">
+                <label class="nhsuk-label nhsuk-checkboxes__label" for="PasswordProtected">
+                  Password protected
+                </label>
+              </div>
+
+              <div class="@passwordCss" id="@passwordId">
+                <div class="nhsuk-form-group">

```

Comment:
Why do we need this div? the component contains a nhsuk-form-group div

Reply:
removed :)